lib/rom/relation.rb

```diff
@@ -189,6 +191,11 @@ def repository
       self.class.repository
     end
 
+    # @api public
+    def to_lazy(*args)
+      Lazy.new(self, *args)
+    end
```

Comment:
I decided on `to_lazy` because that's part of the relation interface which may conflict with app's code. ie somebody would like to define `Users#lazy` returning all the lazy people from the database :)